scantools/run_navvis_to_capture.py

```diff
@@ -213,10 +213,12 @@ def run(input_path: Path, capture: Capture, tiles_format: str, session_id: Optio
         freq_khz = measurement.center_channel_freq_khz
         rssi_dbm = measurement.signal_strength_dbm
         time_offset_us = int(measurement.time_offset_ms) / 1_000
+        ssid = measurement.ssid
```

Comment:
locals like these do not bring much value?

Reply:
I know.. was rather following the pattern